### DISCUSSION TITLE
Vitest globals without imports

### DISCUSSION THREAD

**Discussion by websolutions-hamburg** [vitest-dev/vitest#2951] [Q&A]
Hello,

I use the globals configuration from here: https://vitest.dev/config/#globals
With `tsconfig.json` types. But my IDE (IntelliJ) will still automatically import the globals like:

```TypeScript
import { describe, expect, test } from "vitest";
```

I only found a solutions with the `unplugin-auto-import` package. Is there no other way, to skip the imports of the globals? In Jest it works without any helper package.

**Comment by Tomassito** [comment]
Interesting, as I have exactly opposite experience:
no problem using globals **but** IDE (WebStorm or VC) having troubles with finding these functions ( describe, expect, test ) to import. 

In order to make it work I had to add this path in `tsconfig.app.json` :
`"types": ["vitest/dist/index.d.ts"]`
